examples/classdiagram/class-diagram.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>sprotty Class Diagram Example</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/balloon-css/0.5.0/balloon.min.css">
```

Comment:
Do we need a CQ for that?

Reply:
Hm, I'm really not very firm on these rules, especially when it comes to "just" referencing libraries... :/ Should I ask the Eclipse IP team?